BuildLoader.py

```diff
@@ -1096,7 +1096,7 @@ def pre_build(self):
             if getattr(self._board, "GetKeyHashList", None):
                 key_hash_list = self._board.GetKeyHashList ()
         else:
-            self._board.VERIFIED_BOOT_HASH_MASK = 0
+            self._board.VERIFIED_BOOT_STAGE_1B = 0

```

Comment:
Is this required?

Reply:
Currently we check only PcdVerifiedBootStage1B in stage1A.
Yes, we could remove VERIFIED_BOOT_STAGE_1B  from this location and add "verified boot" check while loading stage1B.

Reply:
Done.